pkg/puller/puller.go

```diff
@@ -29,7 +29,7 @@ const loggerName = "puller"
 
 var errCursorsLength = errors.New("cursors length mismatch")
 
-const DefaultSyncSleepDur = time.Minute
+const DefaultSyncErrorSleepDur = time.Second * 5
```

Comment:
So I thought this should be higher than the kademlia trigger timeout right?... If not this can just keep trying multiple times before it can stop no?

Reply:
kademlia subscription fires whenever there is any change in the depth or topology, so a peer connecting/disconnecting or even storage radius moving around triggers it, so it's unpredictable. 

the kademlia timer triggers the manage loop to attempt new connections.

the original was never to have a higher timeout, as they are not related.